src/_data/catalog/widgets.json

```
@@ -169,6 +169,17 @@
     "link": "https://api.flutter.dev/flutter/widgets/AnimatedWidgetBaseState-class.html",
     "image": "<img alt='' src='/assets/images/docs/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "App Bar",
```

Nit: for names, use the name/capitalization from the m3 site

```suggestion
    "name": "Top app bar",
```

---

`AppBar`'s API doc doesn't mention "top app bar" yet. Are their plans to go through the API docs of M3 widgets and make sure they point out any terminology changes?

---

Happy to change to 'App bar' for capitalization matching, and have updated the others too. 

I'll keep it as 'App bar' - as Tao said, Flutter docs don't define a `TopAppBar`.

I've mapped the widget to the API page as best as I can - for example, Common Buttons links to the `ButtonStyle` class, since it gives the most amount of information about new Material 3 buttons and how to use them effectively.

---

> Common Buttons links to the ButtonStyle class

This would work, I think. However, the M2 widget catalog have a card for each type of button. If we have assets for each button type in M3, maybe it's better to have separate button cards?